extensions/cargo/PhylumExt.toml

```diff
@@ -0,0 +1,10 @@
+name = "cargo"
+description = "Cargo package manager hooks"
+entry_point = "main.ts"
+
+[permissions]
+write = ["./", "~/.cargo", "~/.local/share/phylum/extensions/cargo"]
```

Comment:
I've added an extension function to retrieve the extensions dir, but had to hardcode the permissions here. In theory we could just hardcode it in both places and skip the update to our extension interface?

Alternatively we could also give every extension read/write access to its own extension directory by default.

Reply:
> Alternatively we could also give every extension read/write access to its own extension directory by default.

🤔 Would this allow an extension to change its own permissions without user interaction by writing to its own installed `PhylumExt.toml`?

But `deno` also has `deny_write` now. So we could allow the folder, but deny writing to `PhylumExt.toml`.

Reply:
> 🤔 Would this allow an extension to change its own permissions without user interaction by writing to its own installed PhylumExt.toml?

Yes it would, I forgot we don't prompt for installed extensions.